prometheus/README.md

```diff
@@ -0,0 +1,48 @@
+# Example Usage
+
+`main.go`:
+
+```
+package main
+
+import (
+        "fmt"
+        "github.com/uber-go/tally/prometheus"
+        "math/rand"
+        "net/http"
+        "time"
+)
+
+func main() {
+        r := prometheus.NewReporter()
+
+        r.RegisterCounter("test_counter", map[string]string{"labela": "foo"}, "This is a counter")
+        r.RegisterGauge("test_gauge", map[string]string{"labela": "foo"}, "This is a gauge")
+        r.RegisterTimer("test_histogram", map[string]string{"labela": "foo"}, "This is a histogram", nil)
+
+        go func() {
+                for {
+                        r.ReportCounter("test_counter", map[string]string{"labela": "foo"}, 1)
+                        time.Sleep(1000000)
+                }
+        }()
+        go func() {
+                for {
+                        r.ReportGauge("test_gauge", map[string]string{"labela": "foo"}, rand.Float64()*1000)
+                        time.Sleep(1000000)
+                }
+        }()
+
+        go func() {
+                for {
+                        r.ReportTimer("test_histogram", map[string]string{"labela": "foo"}, time.Duration(rand.Float64()*1000*1000))
+                        time.Sleep(1000000)
+                }
+        }()
```

Comment:
Should we update the example to show using a tally scope to report into it instead perhaps?